static/src/js/app/components/quick-add-button/QuickAddVocabForm.vue

```diff
@@ -295,12 +315,27 @@
         });
         return formatedNodes;
       },
+      // gives access to the state store of forms
+      stateForms() {
+        const { forms } = this.$store.state;
+        return forms;
+      },
       showLatticeNodeList() {
```

Comment:
Note that this can probably be removed now as well as the `latticeNode()` method above.

Reply:
nixed!